### PR TITLE
Describe predefined states comboboxes

### PR DESCRIPTION
**Pull request by davetcoleman**
Replaced https://github.com/ros-planning/moveit_tutorials/pull/434

Addresses @felixvd feedback. Resolved merge conflicts. 

Original author: @takarakasai

### CODE REVIEW

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst [hunk 1]
```diff
@@ -161,6 +161,18 @@ Step 4: Use Motion Planning with the Panda
...
    :width: 700px
 
+Choosing Specific Start/Goal States
++++++++++++++++++++++++++++++++++++
+
+The ``Planning`` tab lets you choose start and goal states for your robot from these options:
+  - The current state
+  - The previous state
+  - A randomly sampled state
+  - Or a named states of the selected planning group, as defined in the ``.srdf`` file of the robot.
```
Comment (felixvd) [inline doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst:171]:
```suggestion
  - Or a named state of the selected planning group, as defined in the ``.srdf`` file of the robot.
```

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst [hunk 2]
```diff
@@ -161,6 +161,18 @@ Step 4: Use Motion Planning with the Panda
 .. image:: rviz_plugin_planned_path.png
    :width: 700px
 
+Choosing Specific Start/Goal States
++++++++++++++++++++++++++++++++++++
+
+The ``Planning`` tab lets you choose start and goal states for your robot from these options:
```
Comment (felixvd) [inline doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst:167]:
```suggestion
The ``Planning`` tab lets you choose start and goal states for a planning request for your robot from these options:
```

File: doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst [hunk 3]
```diff
@@ -161,6 +161,18 @@ Step 4: Use Motion Planning with the Panda
...
+  - The current state
+  - The previous state
+  - A randomly sampled state
+  - Or a named states of the selected planning group, as defined in the ``.srdf`` file of the robot.
+The ``previous`` state here refers to the start state of the previous planning attempt. Thus, selecting ``previous`` as the goal state after execution of a planned motion path allows you to move back to the previous robot pose easily. As, both the ``current`` and the ``previous`` start/goal states are automatically updated after execution, you can easily move back and forth between two states as illustrated in the animation below.
```
Comment (felixvd) [inline doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst:172]:
```suggestion
``Previous`` refers to the start state of the previous planning attempt. Thus, selecting ``previous`` as the goal state after execution of a planned motion path allows you to move back to the previous robot pose easily. As, both the ``current`` and the ``previous`` start/goal states are automatically updated after execution, you can easily move back and forth between two states as illustrated in the animation below.
```

**Review by felixvd — APPROVED** [review]
LGTM with the suggestions/fixes

**Comment by davetcoleman** [comment]
Fixed:
> WARNING: Definition list ends without a blank line; unexpected unindent.

**Pull request merged**